frontend/tests/utils/selectors.ts

```diff
@@ -1,3 +1,3 @@
-export const getTestID = (id: string): string => `css=[data-testid="${id}"]`;
+export const getTestID = (id: string): string => `[data-testid="${id}"]`;
```

Comment:
Is there a reason this is changed?

Reply:
No significant reason, I just didn't like the way it looked, I will revert it

Reply:
Oh if it works without `css=` then let's keep the change! I thought attribute selector required `css=` 😆